app/Services/SC/ScTrialBookingService.cs

```diff
@@ -252,7 +252,10 @@ private async Task<string> GetTrialEmailBodyAsync()
                     booking.TrialLocationRegistryId
                 ),
                 ResultDate = resultDate,
-                TrialBookingId = booking.TrialBookingId
+                TrialBookingId = booking.TrialBookingId,
+                RegistryContactNumber = ScCoreService.GetRegistryContactNumber(
+                    booking.CaseRegistryId
```

Comment:
Wouldn't they want to call the registry where they are trying to book, not the registry where the case originated? 

i.e. `TrialLocationRegistryId`?

Reply:
That's a good point, I don't know which one they'd need to phone but it would make sense to me for it to be the trial location registry. I'll update 👍